pages/wishlist.html

```diff
@@ -0,0 +1,236 @@
+    />
+
+    <!-- Custom CSS -->
+    <link rel="stylesheet" href="/app.css" />
+    <link rel="stylesheet" href="/css/pages/index.css" />
+  </head>
+
+  <body>
+    <!-- header starts -->
+    <header>
+      <!-- name & logo -->
+      <nav class="container flex justify-c-sb alg-i-ctr px-2 py-1">
+        <div class="navbar-align">
+          <!-- logo -->
+
+          <!-- <i class="fa-solid fa-bat text-red"></i> -->
+
+          <a class="store-name" href="/index.html">
+            <h2 class="f-logofont">🐱‍👤Merch-Kart</h2>
+          </a>
+        </div>
+
+        <!-- search bar -->
+        <form class="navbar-search">
+          <input type="text" placeholder="search for product" />
+          <button type="submit">
+            <i class="fas fa-search"></i>
+          </button>
+        </form>
+
+        <div class="navbar-align">
+            <button id="btn-login" class="navbar-btn-log translate-y translate-y-above">
+                <i class="fa-solid fa-right-to-bracket"></i>
+                Log In
+            </button>
+
+          <a
+            href="/pages/wishlist.html"
+            class="badge-btn mx-2 text-white translate-y"
+          >
+            <i class="fas fa-heart"></i>
+            <span class="badge-count bg-red">0</span>
+          </a>
+
+          <a
+            href="/pages/cart.html"
+            class="badge-btn mr-1 text-white translate-y"
+          >
+            <i class="fas fa-shopping-cart"></i>
+            <span class="badge-count bg-red">0</span>
+          </a>
+        </div>
+      </nav>
+    </header>
+    <!-- header ends -->
+
+    <main class="container main">
+      <h1 class="fs-l my-2 text-centre">My Wishlist</h1>
+
+      <p class="fs-md my-2 text-centre">
+        your wishlist is empty till now, <a class="link" href="#">Shop Now</a>
+      </p>
+
+      <ul class="flex flex-wrap justify-c-sa my-3 gap-1">
+
+
+        <li class="m-1">
+            <div class="card card-vertical  shadow-light w-24">
+                <div class="card-vertical-header">
+                    <button class="ml-auto card-icon-btn ">
+                        <i class="fa-regular fa-heart m-auto"></i>
+                    </button>
+                </div>
+                <img class="card-vertical-image" src="/assets/akatsuki-cloud-cotton-face-mask-india-2-700x700.jpg" alt="">
+                <h1 class="card-head">Akatsuki Mask</h1>
+                <p class="card-text h-7 pb-1">
+                    Akatsuki Face Mask for all beloved Naruto Fans.
+                  </p>
+                
+            
+                <div class="card-vertical-footer">
+                    <span class="price">₹139.00</span>
+                    <span class="discount-price">₹1000.00</span>
+                    <span class="discount-percent">90%</span>
+                </div>
+            <button class="btn btn-add-to-cart fs-s">Add to Cart</button>
+          </li>
```

Comment:
Well Structured Code Easy to Understand . proper naming of classes and Ui aslo looks clean .. Good to merge